webapp/channels/src/sass/layout/_headers.scss

```diff
@@ -361,6 +361,10 @@
         display: flex;
         margin-right: 8px;
         gap: 4px;
+
+        .btn-icon {
+            margin:0;
```

Comment:
I'm surprised that the style checker doesn't complain about this not having a space between them. It's so picky about so many things that this feels like an odd thing for it to miss.

Regardless, that's no reason to block this

Reply:
That is odd. Fixed it anyway.